### DISCUSSION TITLE
Add an option for not formatting SVG tags

### DISCUSSION THREAD

**Discussion by babakfp** [prettier/prettier#14424] [Ideas]
Hey
I'm sick of adding `prettier-ignore` all over the place.

Before:

```HTML
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="w-6 h-6">
	<path stroke-linecap="round" stroke-linejoin="round" d="M8.25 6.75L12 3m0 0l3.75 3.75M12 3v18" />
</svg>
```

```HTML
<svg
	xmlns="http://www.w3.org/2000/svg"
	fill="none"
	viewBox="0 0 24 24"
	stroke-width="1.5"
	stroke="currentColor"
	class="h-6 w-6"
>
	<path
		stroke-linecap="round"
		stroke-linejoin="round"
		d="M8.25 6.75L12 3m0 0l3.75 3.75M12 3v18"
	/>
</svg>
```

This👆 is cool, I'm sure it's useful for some projects, but it's annoying and it's a bunch of nonsense. You don't try to read SVG, you just add it to your page to use it as an icon. So you don't need to format 3 lines into 20 lines. Using `prettier-ignore` solved the problem but it also introduces a new problem. Watch this video: https://uploadb.me/k3twr8o6p8ey.html

Add an option that disables the formatting for SVG files and also fixes that problem that you can see in that video.

**Comment by Nadeera3784** [comment]
Hi @babakfp,
You might be able to use .prettierrc.js for this case. the code will be something like this,

```
module.exports = {
  arrowParens: 'avoid',
  bracketSpacing: false,
  singleQuote: true,
  trailingComma: 'all',
  overrides: [
    {
      files: '*.svg',
      options: {
        parser: 'html',
        htmlWhitespaceSensitivity: 'ignore',
      },
    },
  ],
};

```
